scripts/sct_register_multimodal.py

```diff
@@ -53,162 +54,194 @@ def get_parser(paramregmulti=None):
+                    "other images to the destination image. To apply the warping field to another image, use "
+                    "'sct_apply_transfo'"),
+        formatter_class=SmartFormatter,
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py")
+    )
+
+    mandatory = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        '-i',
+        metavar=Metavar.file,
+        required=True,
+        help="Image source. Example: src.nii.gz"
+    )
+    mandatory.add_argument(
+        '-d',
+        metavar=Metavar.file,
+        required=True,
+        help="Image destination. Example: dest.nii.gz"
+    )
+
+    optional = parser.add_argument_group("\nOPTIONAL ARGUMENTS")
+    optional.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        help="Show this help message and exit."
+    )
+    optional.add_argument(
+        '-iseg',
+        metavar=Metavar.file,
+        help="Segmentation source. Example: src_seg.nii.gz"
+    )
+    optional.add_argument(
+        '-dseg',
+        metavar=Metavar.file,
+        help="Segmentation destination. dest_seg.nii.gz"
+    )
+    optional.add_argument(
+        '-ilabel',
+        metavar=Metavar.file,
+        help="Labels source."
+    )
+    optional.add_argument(
+        '-dlabel',
+        metavar=Metavar.file,
+        help="Labels destination."
+    )
+    optional.add_argument(
+        '-initwarp',
+        metavar=Metavar.file,
+        help="Initial warping field to apply to the source image."
+    )
+    optional.add_argument(
+        '-initwarpinv',
+        metavar=Metavar.file,
+        help=("Initial inverse warping field to apply to the destination image (only use if you wish to generate the "
+              "dest->src warping field)")
+    )
+    optional.add_argument(
+        '-m',
+        metavar=Metavar.file,
+        help=("Mask that can be created with sct_create_mask to improve accuracy over region of interest. This mask "
+              "will be used on the destination image. Example: mask.nii.gz")
+    )
+    optional.add_argument(
+        '-o',
+        metavar=Metavar.file,
+        help="Name of output file. Example: src_reg.nii.gz"
+    )
+    optional.add_argument(
+        '-owarp',
+        metavar=Metavar.file,
+        help="Name of output forward warping field."
+    )
+    optional.add_argument(
+        '-param',
+        metavar=Metavar.list,
+        type=list_type(':', str),
+        help=(f"R|Parameters for registration. Separate arguments with \",\". Separate steps with \":\".\n"
+              f"Example: step=1,type=seg,algo=slicereg,metric=MeanSquares:step=2,type=im,algo=syn,metric=MI,iter=5,"
+              f"shrink=2\n"
+              f"  - step: <int> Step number (starts at 1, except for type=label).\n"
+              f"  - type: {{im, seg, imseg, label}} type of data used for registration. Use type=label only at "
```

Comment:
it's a fairly minor issue, but i was wondering if we could go from there:
~~~
                          - step: <int> Step number (starts at 1, except for type=label).
                          - type: {im, seg, imseg, label} type of data used for registration. Use
                          type=label only at step=0.
                          - algo: Default=syn
~~~
to there:
~~~
                          - step: <int> Step number (starts at 1, except for type=label).
                          - type: {im, seg, imseg, label} type of data used for registration. Use
                            type=label only at step=0.
                          - algo: Default=syn
~~~

Reply:
I bet this is tricky. It would add a special case to the indentation calculator I wrote for `R|` strings. Best to save for a separate small PR.

Reply:
A fix for this could also cover a fix for https://github.com/neuropoly/spinalcordtoolbox/pull/2842#issuecomment-689129919 if we use list formatting there, too, e.g.:

```
  -p <str>              Process to generate mask.
                          - <coord,XxY>: Center mask at the X,Y coordinates. (e.g. "coord,20x15")
                          - <point,FILE>: Center mask at the X,Y coordinates of the label defined in
                            input volume FILE. (e.g. "point,label.nii.gz")
                          - <center>: Center mask in the middle of the FOV (nx/2, ny/2).
                          - <centerline,FILE>: At each slice, the mask is centered at the spinal cord
                            centerline, defined by the input segmentation FILE. (e.g.
                            "centerline,t2_seg.nii.gz")
```

Reply:
Since @cfhammill created a separate issue for this (#2889), I'm going to mark the identation comments as resolved.